src/EnergyPlus/HVACCooledBeam.cc

```diff
@@ -735,7 +735,7 @@ namespace HVACCooledBeam {
                         } else {
                             DesCoilLoad = CpAir * RhoAir * DesAirVolFlow *
                                           (state.dataSize->TermUnitFinalZoneSizing(state.dataSize->CurTermUnitSizingNum).DesCoolCoilInTempTU -
-                                           state.dataSize->ZoneSizThermSetPtHi(state.dataSize->CurZoneEqNum));
+                                           state.dataSize->TermUnitFinalZoneSizing(state.dataSize->CurTermUnitSizingNum).ZoneSizThermSetPtHi);
```

Comment:
Moving some of the loose arrays into the `TermUnitFinalZoneSizing` struct caused a bunch of changes like this all over the HVAC code.